### PR TITLE
Update Azerbaijani locale

### PR DESCRIPTION
**Pull request by NARKOZ**

### CODE REVIEW

File: rails/locale/az.yml [hunk 1]
```diff
@@ -116,13 +120,16 @@ az:
...
+      model_invalid: "Yoxlama uğursuz oldu: %{errors}"
       not_a_number: rəqəm deyil
       not_an_integer: tam rəqəm olmalıdır
       odd: tək olmalıdır
+      required: mövcud olmalıdır
       taken: artıq mövcuddur
       too_long: çox uzundur (%{count} simvoldan çox olmalı deyil)
       too_short: çox qısadır (%{count} simvoldan az olmalı deyil)
       wrong_length: uzunluqu səhvdir (%{count} simvol olmalıdır)
+      other_than: "%{count}-dən fərqli olmalıdır"
```
Comment (marmeladze) [inline rails/locale/az.yml]: @NARKOZ as you already know, the ablative case is expressed with two suffixes (_dan_  and _dən_) according to the last syllabus of the word. For numbers, like 6 (_az:_ altı), 9 (_az_: doqquz) etc. the correct ablative case should be formed by _-dan_.

File: rails/locale/az.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ az:
     errors:
       messages:
         record_invalid: 'Yoxlama uğursuz oldu: %{errors}'
+        restrict_dependent_destroy:
+          has_one: "Əlaqəli %{record} yazı olduğundan bu yazını silmək mümkün deyil"
+          has_many: "Əlaqəli %{record} yazılar olduğundan bu yazını silmək mümkün deyil"
```
Comment (marmeladze) [inline rails/locale/az.yml]: @NARKOZ none of turkic languages, including azerbaijani, accept pluralization suffix if already prefixed with numbers. Ex: 1 yazı, (_en: one record_), 23 yazı (_twenty three record**s**_),  42 yazı (_fourty two record**s**_)

File: rails/locale/az.yml (hunk 1 above)
Comment (NARKOZ) [reply rails/locale/az.yml]: Can you provide alternative translation without the suffix?

Comment (marmeladze) [reply rails/locale/az.yml]: Yes, please [review this](https://github.com/svenfuchs/rails-i18n/pull/789). 

P.S. I were not able to generate a meaningful message for `other_than`

**Pull request merged**